### PR TITLE
chore(react-combobox): adopt custom JSX pragma

### PR DESCRIPTION
**Pull request by bsunderhus**
## New Behavior

1. Adopts `react-jsx-runtime` custom pragma on `react-combobox`

### CODE REVIEW

**Comment by fabricteam** [comment]
## [Perf Analysis (`@fluentui/react-components`)](https://github.com/microsoft/fluentui/wiki/Perf-Testing)

  <table>
  <tr>
    <th>Scenario</th>
    <th>Render type</th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">Master Ticks</a>
    </th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">PR Ticks</a>
    </th>
    <th>Iterations</th>
    <th>Status</th>
  </tr><tr>
            <td>InfoButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27608/perf-test-react-components/InfoButton-mount_base.html">21</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27608/perf-test-react-components/InfoButton-mount.html">20</a></td>
            <td>5000</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27608/perf-test-react-components/InfoButton-mount.regression.txt">Possible regression</a></td>
           </tr></table><details><summary>All results</summary><p>
  <table>
  <tr>
    <th>Scenario</th>
    <th>Render type</th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">Master Ticks</a>
    </th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">PR Ticks</a>
    </th>
    <th>Iterations</th>
    <th>Status</th>
  </tr><tr>
            <td>Avatar</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27608/perf-test-react-components/Avatar-mount_base.html">713</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27608/perf-test-react-components/Avatar-mount.html">704</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Button</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27608/perf-test-react-components/Button-mount_base.html">398</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27608/perf-test-react-components/Button-mount.html">387</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Field</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27608/perf-test-react-components/Field-mount_base.html">1294</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27608/perf-test-react-components/Field-mount.html">1333</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProvider</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27608/perf-test-react-components/FluentProvider-mount_base.html">1274</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27608/perf-test-react-components/FluentProvider-mount.html">927</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27608/perf-test-react-components/FluentProviderWithTheme-mount_base.html">113</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27608/perf-test-react-components/FluentProviderWithTheme-mount.html">120</a></td>
            <td>10</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>virtual-rerender</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27608/perf-test-react-components/FluentProviderWithTheme-virtual-rerender_base.html">99</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27608/perf-test-react-components/FluentProviderWithTheme-virtual-rerender.html">99</a></td>
            <td>10</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>virtual-rerender-with-unmount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27608/perf-test-react-components/FluentProviderWithTheme-virtual-rerender-with-unmount_base.html">107</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27608/perf-test-react-components/FluentProviderWithTheme-virtual-rerender-with-unmount.html">115</a></td>
            <td>10</td>
            <td></td>
           </tr>
<tr>
            <td>InfoButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27608/perf-test-react-components/InfoButton-mount_base.html">21</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27608/perf-test-react-components/InfoButton-mount.html">20</a></td>
            <td>5000</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27608/perf-test-react-components/InfoButton-mount.regression.txt">Possible regression</a></td>
           </tr>
<tr>
            <td>MakeStyles</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27608/perf-test-react-components/MakeStyles-mount_base.html">1152</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27608/perf-test-react-components/MakeStyles-mount.html">1131</a></td>
            <td>50000</td>
            <td></td>
           </tr>
<tr>
            <td>Persona</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27608/perf-test-react-components/Persona-mount_base.html">2050</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27608/perf-test-react-components/Persona-mount.html">1994</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SpinButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27608/perf-test-react-components/SpinButton-mount_base.html">1508</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27608/perf-test-react-components/SpinButton-mount.html">1567</a></td>
            <td>5000</td>
            <td></td>
           </tr></table></p></details>

<div id="perfComment9425"/>

**Comment by fabricteam** [comment]
## 📊 Bundle size report

| Package & Exports                                                                                                             |     Baseline (minified/GZIP) |                           PR |                                                                                                                                                                                                 Change |
| :---------------------------------------------------------------------------------------------------------------------------- | ---------------------------: | ---------------------------: | -----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------: |
| <samp>react-combobox</samp> <br /> <abbr title='bundle-size/Combobox.fixture.js'>Combobox (including child components)</abbr> | `89.878 kB`<br />`28.552 kB` | `89.888 kB`<br />`28.575 kB` | `10 B` <br />`23 B`  |
| <samp>react-combobox</samp> <br /> <abbr title='bundle-size/Dropdown.fixture.js'>Dropdown (including child components)</abbr> | `88.562 kB`<br />`28.275 kB` | `88.572 kB`<br />`28.298 kB` | `10 B` <br />`23 B`  |

<details>
<summary>Unchanged fixtures</summary>

| Package & Exports                                                                                                                                                             |          Size (minified/GZIP) |
| ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ----------------------------: |
| <samp>react-components</samp> <br /> <abbr title='bundle-size/ButtonProviderAndTheme.fixture.js'>react-components: Button, FluentProvider & webLightTheme</abbr>              |  `68.803 kB`<br />`18.728 kB` |
| <samp>react-components</samp> <br /> <abbr title='bundle-size/MultipleComponents.fixture.js'>react-components: Accordion, Button, FluentProvider, Image, Menu, Popover</abbr> | `208.156 kB`<br />`57.857 kB` |
| <samp>react-components</samp> <br /> <abbr title='bundle-size/ProviderAndTheme.fixture.js'>react-components: FluentProvider & webLightTheme</abbr>                            |   `38.886 kB`<br />`12.41 kB` |
| <samp>react-portal-compat</samp> <br /> <abbr title='bundle-size/PortalCompat.fixture.js'>PortalCompatProvider</abbr>                                                         |    `6.446 kB`<br />`2.185 kB` |

</details>
<sub>🤖 This report was generated against <a href='https://github.com/microsoft/fluentui/commit/af59ca53d4dfa7e6d80f37002ddcd4107d32b8fa'>af59ca53d4dfa7e6d80f37002ddcd4107d32b8fa</a></sub>
<div id="bundleSizeComment9423"/>

**Comment by fabricteam** [comment]
## 🕵 fluentuiv9 No visual regressions between this PR and main
<div id="vrtCommentfluentuiv9"/>

File: packages/react-components/react-combobox/package.json [hunk 1]
```diff
@@ -41,6 +41,7 @@
     "@fluentui/react-shared-contexts": "^9.3.3",
     "@fluentui/react-theme": "^9.1.7",
     "@fluentui/react-utilities": "^9.8.0",
+    "@fluentui/react-jsx-runtime": "9.0.0-alpha.1",
```
Comment (khmakoto) [inline packages/react-components/react-combobox/package.json:44]: _nit:_ Can you move this above `@fluentui/react-portal` in line `39` to keep alphabetical order?

**Review by khmakoto — APPROVED** [review]

**Pull request merged**